babel.config.js

```diff
@@ -1,8 +1,33 @@
+const path = require('path');
+
 module.exports = {
 	presets: ['@babel/preset-env', '@babel/preset-react'],
 	plugins: [
 		'@babel/plugin-proposal-class-properties',
 		'babel-plugin-codegen',
 		'@babel/plugin-transform-runtime',
+		'macros',
+	],
+
+	overrides: [
+		{
+			test: x => x && x.includes('field-content'),
+			presets: [
+				[
+					'@babel/preset-env',
+					{
+						targets: {
+							node: 10,
+							browsers: [
+								'last 2 chrome versions',
+								'last 2 firefox versions',
+								'last 2 safari versions',
+								'last 2 edge versions',
```

Comment:
We already have this here: https://github.com/WestpacGEL/GEL/blob/master/.browserlistrc

Reply:
And the above is wrong too :)

Reply:
@mitchellhamilton do you have more insights here?

Reply:
This can be removed now because we've removed the copy of the field-content package